app/src/main/java/com/door43/translationstudio/core/TargetLanguage.java

```diff
@@ -65,6 +65,25 @@ public static TargetLanguage generate(JSONObject json) throws JSONException {
         );
     }
 
+    /**
+     * Formats the object as json
+     * @return
+     * @throws JSONException
+     */
+    public JSONObject toApiFormatJson() {
+        try {
+            JSONObject json = new JSONObject();
+            json.put("ld", this.direction.toString());
+            json.put("lc", this.code);
+            json.put("ln", this.name);
+            json.put("lr", this.region);
+            return json;
+        } catch (Exception e) {
+            return null;
```

Comment:
It's not safe to completely ignore an exception. You should always at least `e.printStackTrace()` so that other developers have an opportunity to find the exception in the console if needed.